src/libs/ReportUtils.ts

```diff
@@ -1839,7 +1839,7 @@ function canEditMoneyRequest(reportAction: OnyxEntry<ReportAction>, fieldToEdit
  * Checks if the current user can edit the provided property of a money request
  *
  */
-function canEditFieldOfMoneyRequest(reportAction: OnyxEntry<ReportAction>, reportID: string, fieldToEdit: ValueOf<typeof CONST.EDIT_REQUEST_FIELD>): boolean {
+function canEditFieldOfMoneyRequest(reportAction: OnyxEntry<ReportAction>, reportID: string, fieldToEdit: ValueOf<typeof CONST.EDIT_REQUEST_FIELD>, transaction: Transaction): boolean {
```

Comment:
Shouldn't the transaction be optional here?

Reply:
@DylanDylann Can you check on this?

Reply:
I will check it today

Reply:
@abdulrahuman5196 I just added the optional to transaction param.